lib/cdo/graphics/certificate_image.rb

```diff
@@ -15,31 +15,66 @@ def create_certificate_image2(image_path, name, params={})
   pointsize = 68
   x_offset = params[:x] || 0
   y_offset = params[:y] || 0
-  apply_text(background, name, pointsize, font, color, x_offset, y_offset)
+  apply_text(background, name, pointsize, font, color, x_offset, y_offset, 800, 80)
```

Comment:
What are 800 & 80?  Could we have a comment, or put them in variables with meaningful names before using them?

Reply:
done